gradle/libs.versions.toml

```diff
@@ -42,6 +42,7 @@ fragment = "1.6.1"
 hamcrest = "1.3"
 emergeGradlePlugin = "4.0.0"
 emergeSnapshots = "1.3.0"
+constraintlayout = "1.1.0"
```

Comment:
Ended up adding the constraint layout library since it simplified laying out the elements in the timeline quite a bit, specially considering we need both the `Title` and `TitleAndDescription` alignments. This seems like it increased the size of the library quite a bit, but if users are minifying their app, most of it should be removed (We probably should change emerge tests to run on a minified build)